fract4d/c/fract4dc/arenas.cpp

```diff
@@ -79,7 +79,7 @@ arena_t arena_fromcapsule(PyObject *p)
     arena_t arena = (arena_t)PyCapsule_GetPointer(p, OBTYPE_ARENA);
     if (NULL == arena)
     {
-        fprintf(stderr, "%p : AR : BAD\n", p);
+        fprintf(stderr, "%p : AR : BAD\n", reinterpret_cast<void *>(p));
```

Comment:
Avoid reinterpret_cast<>() - it's basically almost always UB. In this case, it looks like you can static_cast<>() which will still fix the warning but not err on invoking Undefined Behaviour

Reply:
done! thanks for the feedback